### PR TITLE
MM-22267: line-breaks in inline images are now correctly rendered

### PR DESCRIPTION
**Pull request by michelengelen**
#### Summary
inline images added with markdown would not render new lines when there was only a single cr (carriage-return), lf (line-feed), or cr-lf combination present.
This PR adds a new replacer (when markdown inline images are present) that replaces occurrences with closing and opening <p> tags

#### Ticket Link
Fixes [MM-22267](https://mattermost.atlassian.net/browse/MM-22267)

#### Screenshots
Screenshot 2021-02-09 at 16 58 41

### CODE REVIEW

**Review by matthewbirtch — APPROVED** [review]
Works as I would expect now. Thanks @michelengelen

**Comment by michelengelen** [comment]
hey @matthewbirtch could you please re-check? I had to alter the regex for replacing returns a bit and I would like to be sure that it meets the expectation!

File: package-lock.json [hunk 1]
```diff
@@ -12820,7 +12820,10 @@
       "version": "1.3.1",
       "resolved": "https://registry.npmjs.org/full-icu/-/full-icu-1.3.1.tgz",
       "integrity": "sha512-VMtK//85QJomhk3cXOCksNwOYaw1KWnYTS37GYGgyf7A3ajdBoPGhaJuJWAH2S2kq8GZeXkdKn+3Mfmgy11cVw==",
-      "dev": true
+      "dev": true,
```
Comment (devinbinnie) [inline package-lock.json:12823]: Can we revert these `package-lock.json` changes? They aren't needed for this change

**Review by devinbinnie — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (michelengelen) [reply package-lock.json:12823]: sure

**Review by matthewbirtch — APPROVED** [review]
Not seeing any issues. Looks good to me and works as I expect.

**Comment by jgilliam17** [comment]
@michelengelen When comparing to customer reported image, it seems that free line is also not respected.
Screen Shot 2021-02-10 at 10 41 14 AM

**Comment by michelengelen** [comment]
> @michelengelen When comparing to customer reported image, it seems that free line is also not respected.
> Screen Shot 2021-02-10 at 10 41 14 AM

@jgilliam17 this is the behaviour from before the patch ... all new lines were removed from the message. 
I took these screenshots from the current master branch:

**message as in input-field**
Screenshot 2021-02-11 at 10 44 16

**message as rendered**
Screenshot 2021-02-11 at 10 44 23

**Review by jgilliam17 — APPROVED** [review]
Thanks for checking on the free lines issue @michelengelen, apologize about false alarm 🙂 
Tested, looks good to merge.
- Verified line breaks correctly render when inline image is present.

**Review by nevyangelova — APPROVED** [review]

**Pull request merged**